test/index.js

```diff
@@ -379,6 +381,18 @@ describe('Memory', function () {
         done();
     });
 
+    it('cleans up timers when stopped', function (done) {
+
+        var start = Date.now();
+        var child = ChildProcess.fork(Path.join(__dirname, 'fixtures', 'timers'));
+        child.on('exit', function () {
+
+            var stop = Date.now();
+            expect(stop - start).to.be.lessThan(500);
```

Comment:
It seems like this could potentially be flakey on a busy CI system. Can you come up with a different way to test this - preferably without needing to spawn a child process.